<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharPanelAbility</class>
 <widget class="QWidget" name="CharPanelAbility">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
	background-color: white;
	padding: 2 2 2 2;
}</string>
  </property>
  <widget class="MyLabel" name="abilitys_Text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="abilitys_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>181</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{background-color: rgba(255, 255, 255, 0);}
QListWidget::item{background-color: rgba(233, 233, 233, 220);}
QListWidget::item:selected{background-color: rgba(50, 150, 255, 220);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
  </widget>
  <widget class="QListWidget" name="abilitys_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>191</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{background-color: rgba(255, 255, 255, 0);}
QListWidget::item{background-color: rgba(233, 233, 233, 220);}
QListWidget::item:selected{background-color: rgba(50, 150, 255, 220);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
  </widget>
  <widget class="QListWidget" name="abilitys_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>191</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Sans Serif</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget{background-color: rgba(255, 255, 255, 0);}
QListWidget::item{background-color: rgba(233, 233, 233, 220);}
QListWidget::item:selected{background-color: rgba(50, 150, 255, 220);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>MyLabel.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
